migrations/20190328115506_add_asru_establishment_table.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('asru_establishment')
+    .createTable('asru_establishment', table => {
+      table.integer('establishment_id').references('id').inTable('establishments').notNull();
+      table.uuid('profile_id').references('id').inTable('profiles').notNull();
+      table.dateTime('deleted');
```

Comment:
we're going to want to add in `table.timestamps(false, true);` here - it will be useful to know when a relationship is created/updated. Also we reference them in the Model below.

Reply:
done